### DISCUSSION TITLE
Open for new maintainers?

### DISCUSSION THREAD

**Discussion by cwayne18** [kubernetes/dns#660] [Q&A]
Are you all open to adding some maintainers?  Some folks on our team would love to help get this a bit up to date!

**Comment by DamianSawicki** [comment]
> Are you all open to adding some maintainers?

Do you mean adding more people to https://github.com/kubernetes/dns/blob/master/OWNERS? I'm not there myself, so cannot answer this, but the way it usually works in Kubernetes or OSS is that you start contributing and gradually gain credibility for your contributions, which is then formally recognised by the community.

> Some folks on our team would love to help get this a bit up to date!

Sounds great, please do so! Everyone is welcome to propose PRs improving the repo :)
